recipes/pybind11_json/all/conanfile.py

```diff
@@ -23,8 +23,8 @@ def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("nlohmann_json/3.11.2")
-        self.requires("pybind11/2.10.4")
+        self.requires("nlohmann_json/[>=3.9 <4]")
+        self.requires("pybind11/[>=2.6 <3]")
```

Comment:
This library:
- is only used by other 3 libs in conancenter, depending on ``pybind11/2.10.1``. So this probably wouldn't create new conflicts
- IT is true that there are newer ``pybind11/2.13.6``, and it would make sense to let ``pybind11_json`` to move to them.
- I think it is better to keep the lower limit here
```suggestion
        self.requires("pybind11/[>=2.10.4 <3]")
```

Reply:
Your comment mentions that you think it's better to keep the lower limit, yet the code change request raises the limit, could you clarify please? Either way is fine with me, though I selected this range simply because old pybind11 versions are available and compatible. I'd let it be the user's choice what they want to combine in the end.